docs/documentation.rst

```diff
@@ -108,12 +108,12 @@ layout, and use the browser's inspector to look for 404s on static assets.
 
 Publish the new MDN Sphinx Theme
 --------------------------------
-To use the new theme in the public documentation,
+To use the new theme in the public documentation:
 
-1. Commit the new template to ``sphinx-theme``, open a pull request
-2. Remove the clone of the ``sphinx-theme`` from your Kuma repo (Docker method)
-3. After review, merge the PR to the master branch of ``sphinx-theme``
-4. Tag and publish a new version of ``mdn-sphinx-theme`` to PyPI_
-5. Update ``requirements/docs.txt`` in ``kuma``, merge to master
+1. Commit the new template to ``sphinx-theme``, open a pull request.
+2. Remove the clone of the ``sphinx-theme`` from your Kuma repo (Docker method.)
```

Comment:
>> If you may cancel that request so I can move the period that would be great.

I'm not sure what you are asking. You should be able to add a new commit to move the period, or close the pull request if you want to start over from scratch.

I think this is an OK change, once the period is moved.